src/xcode/fastlane/Fastfile

```diff
@@ -222,6 +222,12 @@ platform :ios do
       --header 'Authorization: Bearer #{ENV["TSI_CLOUD_TOKEN"]}' \
       --form 'file=@../ENA.ipa'"
     )
+
+    # Trigger Tests on Mobile Device Cloud via CircleCI (cwa-app-automation)
+    sh("curl -u '#{ENV["CIRCLE_API_ADMIN_TOKEN"]}': \
+      -d 'build_parameters[CIRCLE_JOB]=android' \
```

Comment:
I don't know this endpoint but is android really correct here?

Reply:
upps... copy&paste bug... my fault - now it's: `ios`